src/styles/forms.scss

```diff
@@ -28,62 +17,3 @@
   max-width: 100%;
   resize: vertical;
 }
-
-.form-actions {
-  @include forms.pie-clearfix;
-  margin-top: 0.75em;
-}
-
-.btn {
```

Comment:
Hard to grep for! But I tried to be thorough and I am confident there are no remaining uses of this class in either annotator or sidebar source.

Reply:
I couldn't find any references either with `(?<![A-Za-z_-])btn` as the pattern (match `btn` [not preceded by](https://www.regular-expressions.info/refadv.html) a letter, hyphen or dash)